main/test/ca/uwaterloo/flix/library/TestList.flix

```diff
@@ -1023,6 +1023,21 @@ def foldRight03(): Bool = List.foldRight((e, i) -> (i - e)*(e rem 2 + 1), 100, 1
 @test
 def foldRight04(): Bool = List.foldRight((e, i) -> (i - e)*(e rem 2 + 1), 100, 1 :: 2 :: 3 :: Nil) == 382
 
+/////////////////////////////////////////////////////////////////////////////
+// foldRightL                                                              //
+/////////////////////////////////////////////////////////////////////////////
+@test
+def foldRightL01(): Bool = List.foldRightL((e, i) -> (force i - e)*(e rem 2 + 1), 100, Nil) == 100
```

Comment:
space between *

Reply:
I see, I was copying the style from other tests above, should I change them as well?